scripts/encapp_tool/ffutils.py

```diff
@@ -76,15 +76,29 @@ def video_is_y4m(videofile):
     return extension == ".y4m"
 
 
-def get_video_info(videofile, debug=0):
-    assert os.path.exists(videofile), f"input video file {videofile} not exist"
-    assert os.path.isfile(videofile), f"input video file {videofile} is not a file"
-    assert os.access(
-        videofile, os.R_OK
-    ), f"input video file ({videofile}) is not readable"
+def get_video_info(videofile, debug=0, use_lcevc=False):
```

Comment:
So, the situations where the ffmpeg version does not handle lcevc must be handled (I am unable to test this right now because I am on MacOs and the other machine is a Fedora...).
Ffprobe should tell you that this is lcevc.